powerline_shell/segments/svn.py

```diff
@@ -1,29 +1,51 @@
 import subprocess
-from ..utils import BasicSegment, RepoStats
+from ..utils import ThreadedSegment, RepoStats
 
 
-class Segment(BasicSegment):
-    def add_to_powerline(self):
-        is_svn = subprocess.Popen(["svn", "status"],
-                                  stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-        is_svn_output = is_svn.communicate()[1].decode("utf-8").strip()
-        if len(is_svn_output) != 0:
-            return
+class Segment(ThreadedSegment):
+    def __init__(self, powerline):
+        super().__init__(powerline)
```

Comment:
This project currently supports Python 2 and `super()` won't work there. Dropping python 2 has been something on my mind, but right now that's the situation.

Reply:
Does 2fba29e fix this?